COPYING

```diff
@@ -1,7 +1,7 @@
                     GNU GENERAL PUBLIC LICENSE
                        Version 3, 29 June 2007
 
- Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
```

Comment:
This change is invasive.

Reply:
Fixed. Any additional comments?

Reply:
The GNUGPL3 has https: https://www.gnu.org/licenses/gpl-3.0.txt